app/views/layouts/auth.html.slim

```diff
@@ -2,13 +2,21 @@ doctype html
 html
   head
     meta content=("text/html; charset=UTF-8") http-equiv="Content-Type" /
-    title PersonalFinance
+    title= t('personal_finance')
     = csrf_meta_tags
-    = stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track' => 'reload'
-    = javascript_include_tag 'application', 'data-turbolinks-track' => 'reload'
     = javascript_pack_tag 'application'
-    = stylesheet_pack_tag 'bootstrap'
+    = stylesheet_pack_tag 'style'
     = stylesheet_link_tag "https://maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css"
   body.bg-dark
     .container
+      - if notice
+        .alert.alert-success.close aria-label="Close"
+          button.close aria-label="Close" data-dismiss="alert" type="button"
+            span aria-hidden="true"  ×
+          = notice
+      - if alert
+        .alert.alert-danger.close aria-label="Close"
+          button.close aria-label="Close" data-dismiss="alert" type="button"
+            span aria-hidden="true"  ×
+          = alert
```

Comment:
@marderer why didn't you render `views/layouts/_notice.html.slim` here instead of duplicating logic?